bootstrapping_decision_making.md

```diff
@@ -28,3 +28,5 @@ This framework has the following characteristics:
 
 - By gradually increasing the size of the decision making body one person at a time, the power implicit in this election process is gradually distributed over a larger group of people.
 - At each stage where the new decision making body is considering adding a new person, they can also choose to stop electing additional members. This allows the group to decide on the overall size in an organic, gradual manner.
+- In the spirit of Jupyter always seeking to develop multi-stakeholder governance, this bootstrapping process should try to maximize the participation of individuals from multiple organizations. Specifically, they should when possible alternate across organizations to avoid all initial members of the decision making body belonging to a single organization.
+- As explained in the SSC document, there will be a working group with an SSC representative focused on issues of Inclusive Community Development. This representative can be a point of contact to assist with any issue regarding inclusion and participation in the creation of the project decision making bodies.
```

Comment:
I believe that this would be better as a standing working group of the board of directors. The chair of the working group will also have representation as a member of the SSC.

Reply:
Yup, that's actually the plan! Sorry if it wasn't too clear, but the BoD PR (that we'll try to open as soon as we complete work on this one) will list that as a necessary WG under the Board of Directors. I think in the end we could have chairs delegate representation to the SSC at their discretion, but that's a minor detail that can be worked later. I think your key point is one we're already in agreement on :)

Reply:
In my experience “diversity and inclusion” working groups are often marginalized within organizations. They exist as a token of D&I but do not have actual power to create change. What are ways that the Jupyter D&I group will have tangible authority over sub projects that do not meet Jupyter standards of D&I?